lib/routes/routePUT.js

```diff
@@ -120,6 +120,14 @@ export default function routePUT(request, response, log, utapi) {
         // PUT object copy
         // if content-md5 is not present in the headers, try to
         // parse content-md5 from meta headers
+
+        if (request.headers['content-md5'] === '') {
```

Comment:
its apply only if content-md5 is set and its an empty string, so i guess its ok. But do we have other case where we should return InvalidDigest ? like a malformed md5 ?